recipes-kernel/linux/linux-smelt/defconfig

```diff
@@ -3531,31 +3464,3 @@ CONFIG_ARCH_HAS_ATOMIC64_DEC_IF_POSITIVE=y
 CONFIG_QMI_ENCDEC=y
 # CONFIG_QMI_ENCDEC_DEBUG is not set
 # CONFIG_VIRTUALIZATION is not set
-CONFIG_IP_NF_MULTIPLE_TABLES=y
-CONFIG_CGROUP_MEM_RES_CTLR=y
-CONFIG_CGROUP_MEM_RES_CTLR_SWAP=y
-CONFIG_CGROUP_MEM_RES_CTLR_KMEM=y
-CONFIG_BLK_SCHED=y
```

Comment:
I don't find the documentation for this but I imagine this is a downstream block scheduler, is there any reason for disabling it?

Reply:
See previous, this has probably happened when I disabled CGROUP debugging.

Reply:
This is actually enabled by `check_config`, but Google doesn't even result in anything useful :-)

Reply:
That will do